src/validators/mainnet.json

```diff
@@ -481,6 +481,14 @@
       "description": "A web3 professional firm ready to assist you from the get-go. DeFimans is a web3 consulting firm based in Japan with crypto-native members with DeFi and TradFi backgrounds. Our core strength lies in three areas. First is client connection with domestic/global web3 players; we have very close connections to some of Japan's biggest entertainment and finance companies, top-tier DeFi projects, and startups. Next, Unparalleled insight into the Japanese web3 space. Based on our connections and experience in this unique market, we can provide crucial information and networking to assist global projects looking to GTM in Japan or vice versa. Lastly, the highest concentration of DeFi and tokenomics expertise in Japan. We advise tokenomics and capital policy for global DeFi projects and web3 projects of the biggest Japanese companies in the finance and entertainment sectors.",
       "website": "https://defimans.com/en/",
       "twitter": "https://x.com/DeFimans"
+    },
+    {
+      "id": "0xb6380cbd103b20dbde1a47df1d1cc299f01dda8966e10a395e4430ba2871f719529ea5cd9ffa9d48625b0d96dd77fb50",
+      "logoURI": "https://res.cloudinary.com/duv0g402y/image/upload/c_thumb,w_200,g_face/v1750702385/src/validators/mainnet/y7t9f1a7y7slfclwbdsn.png",
```

Comment:
https://github.com/berachain/metadata/pull/254/commits/b4e5188085044940c647477132960fab78aa34be

looks like this was dropped

Reply:
https://res.cloudinary.com/duv0g402y/image/upload/v1748439531/validators/0xb6380cbd103b20dbde1a47df1d1cc299f01dda8966e10a395e4430ba2871f719529ea5cd9ffa9d48625b0d96dd77fb50.png